Add Toolsetter (Long John) guide and restructure for multiple docs

EdwardChamberlain opened the pull request:
Restructures to add space for new products
Adds Long John MD guide

fixes #17 
fixes #8

benagricola committed:
Move milo docs into subdirectory

Signed-off-by: Ben Agricola <ben+git@agrico.la>

EdwardChamberlain committed:
added comming soon info.

EdwardChamberlain committed:
added milo index

EdwardChamberlain committed:
corrected paths

EdwardChamberlain committed:
added reference to the PDF manual

EdwardChamberlain committed:
Merge pull request #3 from MillenniumMachines/refactor-docs-to-support-future-projects

Refactor docs to support future projects

EdwardChamberlain committed:
Merge pull request #4 from MillenniumMachines/main

sync main to dev

EdwardChamberlain committed:
added sourcing and printing

EdwardChamberlain committed:
Merge pull request #6 from MillenniumMachines/long-john-toolsetter-docs

added sourcing and printing

EdwardChamberlain committed:
added lj to nav

EdwardChamberlain committed:
added placeholder files and links

EdwardChamberlain committed:
assembly docs

EdwardChamberlain committed:
remove spacing

EdwardChamberlain committed:
update navbar

EdwardChamberlain committed:
removed FAQ

EdwardChamberlain committed:
updated loctite note.

EdwardChamberlain committed:
added sping info - fixes #8

EdwardChamberlain committed:
Added info on self sourcing - fear not!

EdwardChamberlain committed:
added handwheel images - text required.

EdwardChamberlain committed:
added info

EdwardChamberlain committed:
Put handwheels on mill added

EdwardChamberlain committed:
repeat steps for Y axis added

EdwardChamberlain committed:
corrected codeblock

EdwardChamberlain committed:
Flip image order to correct

EdwardChamberlain committed:
Move electronics into assembly

EdwardChamberlain committed:
Added wiring diagram

EdwardChamberlain committed:
update nav

EdwardChamberlain committed:
moved ts to Z max

EdwardChamberlain committed:
Added basic wiring directions + LDO kit link

benagricola committed:
Merge branch 'main' into dev

benagricola commented:
Merged `main` into `dev` to fix the conflicting images.

benagricola commented on docs/index.md:
```diff
@@ -4,8 +4,20 @@
+- [Electronics Manual](./milo/electronics_manual/index.md)
+
+#### Casa Enclosure - Enclosure for Milo
+Docs comming Soon! For now please use the PDF manual.
```
*Puts on spellcheck hat*: "Docs coming soon!"

benagricola commented on docs/index.md:
```diff
@@ -4,8 +4,20 @@
+Docs comming Soon! For now please use the PDF manual.
+
+#### Fixated - Fixture plate for Milo
+Docs comming Soon! For now please use the PDF manual.
```
Same here

EdwardChamberlain replied:
oops - resolved in 9d0e27d

EdwardChamberlain committed:
spell check: comming -> coming

benagricola commented on docs/long-john/assembly_manual/assembly_manual.md:
```diff
@@ -0,0 +1,5 @@
+# Assembly Manual
+!!! info annotate "Loctite"
```
I switched away from using the term "Loctite" in the assembly manual as it's a brand name (which also sells Superglue!) rather than the product.

The term I went with was "medium-strength thread-lock" and in one place "generally, blue" is mentioned, see [here](https://millenniummachines.github.io/docs/assembly_manual/chapters/y_axis_assembly/#xy-plate-orientation)

I think we should go with "medium-strength thread-lock" when talking about this, but using Loctite 243 or Loctite 248 (I prefer, slightly lower breaking torque and comes in a stick!) as an example would be fine :+1:

EdwardChamberlain replied:
moved to 'threadlocker' consistently

EdwardChamberlain committed:
ignore vscode folder

benagricola committed:
SPAG Attack

Various spelling and grammar fixes, some markdown lint changes and added
a new thread-lock FAQ entry (although this isn't Milo specific so there
might be a better place to put it, but fine here for now).

Signed-off-by: Ben Agricola <ben+git@agrico.la>

benagricola committed:
Remove quotes from red, red is fine

Signed-off-by: Ben Agricola <ben+git@agrico.la>

benagricola committed:
Formatting changes of Milo BOM for consistency

Signed-off-by: Ben Agricola <ben+git@agrico.la>

benagricola reviewed (changes requested):
Can we change the Loctite name to "medium-strength thread-lock" as noted? I doubt many, if any people building a mill will fall foul of picking the "wrong" Loctite but sods law is we would somehow find that one person :sweat_smile: 

Other than that, looks great to me, nice work :+1:

EdwardChamberlain committed:
loctite -> medium-strength thread-lock

EdwardChamberlain committed:
threadlocker consistency

EdwardChamberlain committed:
grammar

EdwardChamberlain committed:
added github links

EdwardChamberlain committed:
remove https:

EdwardChamberlain committed:
readd https

benagricola commented on docs/milo/faq.md:
```diff
@@ -109,11 +109,11 @@ If you're new to milling then you probably want to buy two sets :sweat_smile:
-Medium-strength thread-lockers will usually have a break-loose torque of between 10 and 25 Nm, which is achievable with an allen key in most situations.
+Medium-strength threadlockers will usually have a break-loose torque of between 10 and 25 Nm, which is achievable with an allen key in most situations.
 
 Do not use permanent or semi-permanent thread-lock, or red colour thread-lock unless it is specifically marked as medium strength.
```
```suggestion
Do not use permanent or semi-permanent threadlockers, or red colour threadlockers unless it is specifically marked as medium strength.
```

EdwardChamberlain committed:
Update docs/milo/faq.md

Co-authored-by: Ben Agricola <717328+benagricola@users.noreply.github.com>
Signed-off-by: Ed <edward.chamberlain@me.com>

benagricola committed:
Consistently apply thread-lock -> threadlock to all docs

Signed-off-by: Ben Agricola <ben+git@agrico.la>

benagricola commented:
Last commit just changes `thread-lock` to `threadlock` across the docs so everything should consistently refer to `threadlock`, `threadlocker`, `threadlockers`, or `threadlocking` now
